index.html

```diff
@@ -1,52 +1,634 @@
-<!DOCTYPE HTML>
-<!--
-	Aerial by HTML5 UP
-	html5up.net | @n33co
-	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
--->
-<html>
-	<head>
-		<title>Gabe Benjamin - Software Engineer</title>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1" />
-		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
-		<link rel="stylesheet" href="assets/css/main.css" />
-		<link rel="shortcut icon" type="image/x-icon" href="assets/css/images/favicon.ico">
-		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
-		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
-	</head>
-	<body class="loading">
-		<div id="wrapper">
-			<div id="bg"></div>
-			<div id="overlay"></div>
-			<div id="main">
-
-				<!-- Header -->
-					<header id="header">
-						<h1>Gabe Benjamin</h1>
-						<p>Software Engineer @ <a href="https://www.google.com/">Google</a></p>
-						<nav>
-							<ul>
-								<li><a href="Gabe_Benjamin_Resume.pdf" class="icon fa-file-text"><span class="label">Resume</span></a></li>
-								<li><a href="https://www.linkedin.com/in/gabebenjamin" class="icon fa-linkedin-square"><span class="label">LinkedIn</span></a></li>
-								<li><a href="https://github.com/GabeBenjamin" class="icon fa-github"><span class="label">Github</span></a></li>
-								<li><a href="mailto:GabeKEBenjamin@gmail.com" class="icon fa-envelope-o"><span class="label">Email</span></a></li>
-							</ul>
-						</nav>
-					</header>
-
-				<!-- Footer -->
-					<footer id="footer">
-						<span class="copyright">&copy; Gabe Benjamin. Design: <a href="http://html5up.net">HTML5 UP</a>.</span>
-					</footer>
-
-			</div>
-		</div>
-		<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
-		<script>
-			window.onload = function() { document.body.className = ''; }
-			window.ontouchmove = function() { return false; }
-			window.onorientationchange = function() { document.body.scrollTop = 0; }
-		</script>
-	</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Gabe Benjamin - Senior Software Engineer</title>
+  <meta name="description"
+    content="Gabe Benjamin is a Fullstack Senior Software Engineer with over 5 years of experience and currently works at Google" />
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="assets/css/styles.css" />
+  <link rel="shortcut icon" type="image/x-icon" href="assets/images/favicon.ico" />
+  <link rel="stylesheet" href="https://use.typekit.net/oyr7lzg.css">
+</head>
+
+<body>
+  <div class="background background-top no-shrink">
+    <svg class="top-wave" role="presentation" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1186.65 976.02">
```

Comment:
Consider replacing inline svgs with `<img>` tags pointing to .svg files in the assets/ folder.

https://vecta.io/blog/best-way-to-embed-svg

Reply:
Yeah I embedded them directly because I was thinking it would be cool to do something interactive with them, but I haven't found a good way to do that yet so I'll move them. I appreciate the link with the tradeoffs!